webrtc.html

```diff
@@ -7698,7 +7698,9 @@ <h3 id="sec.identity-verify-assertion">Verifying Identity Assertions</h3>
     session.</p>
 
     <p>Revealing IP addresses can leak location and means of
-    connection; this can be sensitive.</p>
+    connection; this can be sensitive but necessary for establishing peer to
+    peer connections. The choice of which addresses to expose is controlled by
```

Comment:
Would it make sense to change "which addresses to expose" into "which addresses to expose and when to expose them"?